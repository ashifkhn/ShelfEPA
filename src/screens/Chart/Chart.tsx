import React from 'react';
import { View, Text, StyleSheet } from 'react-native';

const Chart= () => {
  return (
    <View style={styles.container}>
      <View style={styles.header}>
        <Text style={styles.headerText}>Expense Chart</Text>
      </View>
      <View style={styles.chartContainer}>
        {/* Add your custom chart component here */}
        <View style={styles.chart}>
          {/* Your chart content goes here */}
          <Text style={styles.chartText}>Your Chart Goes Here</Text>
        </View>
      </View>
      <View style={styles.footer}>
        <Text style={styles.footerText}>Your expense trend over the last six months</Text>
      </View>
    </View>
  );
};

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: 'white',
    padding: 20,
  },
  header: {
    marginTop: 50,
    marginBottom: 20,
  },
  headerText: {
    fontSize: 24,
    color: '#0373F3',
    textAlign: 'center',
  },
  chartContainer: {
    alignItems: 'center',
  },
  chart: {
    borderWidth: 1,
    borderColor: '#0373F3',
    borderRadius: 10,
    padding: 20,
    marginTop: 20,
    alignItems: 'center',
  },
  chartText: {
    fontSize: 16,
    color: '#0373F3',
  },
  footer: {
    marginTop: 20,
    alignItems: 'center',
  },
  footerText: {
    fontSize: 16,
    color: '#0373F3',
    textAlign: 'center',
  },
});

export default Chart;
