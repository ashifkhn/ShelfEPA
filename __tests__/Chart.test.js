import "react-native"
import React from "react"
import renderer from "react-test-renderer"
import Chart from "../src/screens/Chart/Chart";


test('Home renders correctly', () => {
  const tree = renderer.create(<Chart />).toJSON();
  expect(tree).toMatchSnapshot();
});