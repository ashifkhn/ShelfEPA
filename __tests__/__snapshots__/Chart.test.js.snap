// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<View
  style={
    {
      "backgroundColor": "white",
      "flex": 1,
      "padding": 20,
    }
  }
>
  <View
    style={
      {
        "marginBottom": 20,
        "marginTop": 50,
      }
    }
  >
    <Text
      style={
        {
          "color": "#0373F3",
          "fontSize": 24,
          "textAlign": "center",
        }
      }
    >
      Expense Chart
    </Text>
  </View>
  <View
    style={
      {
        "alignItems": "center",
      }
    }
  >
    <View
      style={
        {
          "alignItems": "center",
          "borderColor": "#0373F3",
          "borderRadius": 10,
          "borderWidth": 1,
          "marginTop": 20,
          "padding": 20,
        }
      }
    >
      <Text
        style={
          {
            "color": "#0373F3",
            "fontSize": 16,
          }
        }
      >
        Your Chart Goes Here
      </Text>
    </View>
  </View>
  <View
    style={
      {
        "alignItems": "center",
        "marginTop": 20,
      }
    }
  >
    <Text
      style={
        {
          "color": "#0373F3",
          "fontSize": 16,
          "textAlign": "center",
        }
      }
    >
      Your expense trend over the last six months
    </Text>
  </View>
</View>
`;
